### PR TITLE
✨Fill DetailsPage with data from db.json 

### PR DESCRIPTION
**Pull request by Sina-Sche**
-  [x] access Product-Id via useParams() to display data on DetailPage
       ProductImage, ProductDescription, Price and Link to website

- [x] add Links to DetailPage 

- [x] edit Styling of DetailPage

👀 in the deployment, access the detailPage by clicking on one of the images, either on the home or searchpage 
--> Will work on more responsive design soon - until then please only view my deployment on phone-screen size ;)

### CODE REVIEW

**Comment by kschmitz123** [comment]
wie besprochen 😄
lgtm 🚀

**Review by Naasgul — APPROVED** [review]
lgtm! 👍

**Review by mhstrkmp — APPROVED** [review]
lgtm 🚀

File: client/src/components/Image.js [hunk 1]
```diff
@@ -41,4 +42,5 @@ Image.propTypes = {
   src: PropTypes.string,
   alt: PropTypes.string,
   size: PropTypes.oneOf(["small", "large"]),
+  id: PropTypes.number,
```
[the remark this answers is not in the document]
Comment (Sina-Sche) [reply client/src/components/Image.js:45]: good point! Will add this with the next Pull Request :)

**Pull request merged**